StephaneGrabsia/uQuizz
# Student main page front

This PR continue the development of the main page of the student

This one allows to manage the connection and the deconnection of the student.

To test it manually, you can:
- launch the site manually
- Create a teacher with the register form
- Create a classroom on the endpoint room/create/ while using the id of the previously created teacher (is id should be 1 if you have recreated the db)
- Logging you in the home page by using the classroom id previously created and by giving an username
- Checking if the username printed in the top right corner of the page is the same than the one you gave
- Disconnect yourself with the "Logout" button in the menu present at the top right corner of the page
